app/config/init.js

```diff
@@ -0,0 +1,43 @@
+const vm = require('vm');
+const notify = require('../notify');
+
+const _extract = function (script) {
+  const module = {};
+  script.runInNewContext({module});
+  if (!module.exports) {
+    throw new Error('Error reading configuration: `module.exports` not set');
+  }
+  return module.exports;
+};
+
+const _syntaxValidation = function (cfg) {
```

Comment:
```js
const _syntaxValidation = function (cfg) {
    try {
      return new vm.Script(cfg, { filename: '.hyper.js', displayErrors: true, lineOffset: 0 })
    } catch (error) {
      notify(`Error loading config: ${error.name}, see DevTools for more info`)
      console.error('Error loading config:', error)
      return
    }
}
```
tested:
```js
_syntaxValidation(`
const c = 'a'
const b = 'd'
const c = '2'
`)
```

notify: *Error loading config: SyntaxError, see DevTools for more info*
console:
```js
 Error loading config:
 .hyper.js:4
const c = '2'
          ^^^
SyntaxError: Identifier 'c' has already been declared